docs/spec/relayer/PacketHandlers.tla

```diff
@@ -0,0 +1,125 @@
+--------------------------- MODULE PacketHandlers ---------------------------
+
+(***************************************************************************
+ This module contains definitions of operators that are used to handle
+ packet datagrams
+ ***************************************************************************)
+
+EXTENDS Integers, FiniteSets, RelayerDefinitions    
+
+(***************************************************************************
+ Packet datagram handlers
+ ***************************************************************************)
+
+\* Handle "PacketRecv" datagrams
+HandlePacketRecv(chainID, chain, packetDatagram, log) ==
+    \* get chainID's connection end
+    LET connectionEnd == chain.connectionEnd IN
+    \* get chainID's channel end
+    LET channelEnd == chain.connectionEnd.channelEnd IN
+    \* get packet
+    LET packet == packetDatagram.packet IN
+    
+    IF \* if the channel and connection ends are open for packet transmission
+       /\ channelEnd.state /= "UNINIT"
```

Comment:
In the ICS, the check is whether `channelEnd != null` and `channelEnd.state = OPEN`. So I was trying to capture that with this check, since in the TLA spec, `channelEnd == null` is modeled by `channelEnd.state /= UNINIT`.